assets/js/hooks/local-storage.js

```diff
@@ -42,5 +44,13 @@ export const hooks = {
     }
   },
 
-  resetRoomsToBackup(obj) { localStorage.setItem(obj.key, 0) }
+  resetRoomsToBackup(obj) { localStorage.setItem(obj.key, 0) },
+
+  setupAuthEvents(key) {
+    window.addEventListener('storage', event => {
```

Comment:
When this gets fired ?

Reply:
when local storage has ben modified
https://developer.mozilla.org/en-US/docs/Web/API/Window/storage_event